src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Conformance/ConformanceBuilderTests.cs

```diff
@@ -120,6 +120,18 @@ public void GivenAConformanceBuilder_WhenAddingResourceSearchParam_ThenTypeSearc
             Assert.Null(typeName);
         }
 
+        [Fact]
+        public void GivenAConformanceBuilder_WhenAddingDefaultInteractions_ThenProfileIsAddedAtResource()
+        {
+            _builder.AddDefaultResourceInteractions();
+
+            ITypedElement statement = _builder.Build();
+
+            bool hasProfile = (bool)statement.Scalar($"{ResourceQuery("Account")}.profile.exists()");
+
+            Assert.True(hasProfile);
```

Comment:
should this validate the actual profile value as oppose to only checking if the value exists?